### PR TITLE
[tempo] add single mode

### PR DESCRIPTION
**Pull request by swartz-k**
Hey，all，Based [tempo chart](https://github.com/grafana/tempo/tree/master/operations/helm)，I have add single binary tempo chart

Signed-off-by: swartz-k <9215868@gmail.com>

### CODE REVIEW

**Comment by joe-elliott** [comment]
I have run this locally and it works.  Is there anything keeping this from being merged?

**Comment by stevehipwell** [comment]
Is there a timeframe for this?

**Review by afoninsky — APPROVED** [review]

File: charts/tempo/values.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# -- Overrides the chart's name
+nameOverride: ""
+
+# -- Overrides the chart's computed fullname
+fullnameOverride: ""
+
+replicas: 1
+tempo:
+  repository: grafana/tempo
+  tag: latest
```
Comment (joe-elliott) [inline charts/tempo/values.yaml:10]: please change this to 0.5.0 and then I'll merge.  we should pin versions.

**Review by joe-elliott — CHANGES_REQUESTED** [review]
one change requested and then i'll merge.

**Review by joe-elliott — APPROVED** [review]

**Pull request merged**